### PR TITLE
Battle Panel Multi Resolution

### PR DESCRIPTION
**Pull request by andrzejfalkowski**
Battle panel in different variants for each main vertical resolution and 720 widescreen.

If there's not enough space to display all mercs due to increased limits compared to vanilla, mercs will be displayed page by page, scrollable with Page Up/Page Down (similar to the team assignments panel).

Related fixes:
- some UI elements like sort buttons wouldn't be rendered during animated prebattle interface transition if prebattle dialogue was skipped
- in 720 widescreen it was possible to pick up inventory items behind prebattle interface
- in 720 widescreen autoresolve overlay wasn't rendered on the entire screen

Comparison:

640x480
battle_panel_640_old
battle_panel_640_new

1280x720 Widescreen
battle_panel_720_old
battle_panel_720_new

1024x768
battle_panel_768_old
battle_panel_768_new

720 Widescreen - Auto-resolve fix
auto_battle_720_old
auto_battle_720_new

Transition fix
transition_640_old
transition_640_new

Corresponding gamedir PR:
https://github.com/1dot13/gamedir/pull/46

### CODE REVIEW

**Comment by Asdow** [comment]
This looks great! One thing I did notice, is that it's not taking into account localized versions of prebattlepanel graphics.

For example, in German version, the panel column texts are not replaced by the graphics from Data\german\PREBATTLEPANEL_GERMAN.sti

Current:
jagged_alliance_2-2023-08-20-12_22_24

Pull request:
jagged_alliance_2-2023-08-20-12_20_13

Same thing happens with the other languages as well.

File: Strategic/PreBattle Interface.cpp [hunk 1]
```diff
@@ -482,31 +495,70 @@ void InitPreBattleInterface( GROUP *pBattleGroup, BOOLEAN fPersistantPBI )
 	fMapScreenBottomDirty = TRUE;
 	ChangeSelectedMapSector( gubPBSectorX, gubPBSectorY, gubPBSectorZ );
 	// Headrock: Added FALSE argument, We might need TRUE but not sure. Will need to initiate battle :)
-	RenderMapScreenInterfaceBottom( FALSE );
+	//RenderMapScreenInterfaceBottom( FALSE );
```
Comment (Asdow) [inline Strategic/PreBattle Interface.cpp:498]: If this function call is no longer needed, simply delete it and the accompanying comment related to it. There's plenty of old, commented out code in the repo. We don't need to add to it :)

Comment (andrzejfalkowski) [reply Strategic/PreBattle Interface.cpp]: Cool cool, deleted.

Localization: https://github.com/1dot13/gamedir-languages/pull/10
BTW looks like Dutch PREBATTLEPANEL_DUTCH.STI was in already only in English, so I just added other resolution variants also in English for Dutch in PR.

File: Utils/Multi Language Graphic Utils.cpp [hunk 2]
```diff
@@ -203,6 +212,15 @@ BOOLEAN GetMLGFilename( SGPFILENAME filename, UINT16 usMLGGraphicID )
 			case MLG_PREBATTLEPANEL:
 				sprintf( filename, "GERMAN\\PreBattlePanel_german.sti" );
 				return TRUE;
+			case MLG_PREBATTLEPANEL_800x600:
+				sprintf(filename, "INTERFACE\\PreBattlePanel_800x600_german.sti");
```
Comment (Asdow) [inline Utils/Multi Language Graphic Utils.cpp:216]: The correct path should be GERMAN\\PreBattlePanel_800x600_german.sti, similar to the original prebattle sti file. Right now the game will error out on the assert at line 546 in PreBattle Interface.cpp as it fails to fails to find the file

Comment (andrzejfalkowski) [reply Utils/Multi Language Graphic Utils.cpp]: Thanks, resolved

**Pull request merged**